static/js/treatment.js

```diff
@@ -54,13 +40,8 @@
   function treatmentSubmit(event) {
 
     var data = prepareData();
-    var errors = checkForErrors(data);
 
-    if (errors.length > 0) {
-      window.alert(errors.join('\n'));
-    } else {
-      confirmPost(data);
-    }
+    confirmPost(data);
 
     if (event) {
       event.preventDefault();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (8)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/800382163/issues/source?bid=2452379&fileBranchId=2725437#l51)